packages/icons/src/library/connection.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path, G } from '@wordpress/primitives';
+
+const connection = (
+	<SVG
+		width="24"
+		height="24"
+		viewBox="0 0 24 24"
+		xmlns="http://www.w3.org/2000/svg"
+	>
+		<Path d="M5 19L8 16L5 19Z" fill="#9747FF" />
+		<Path d="M16 8L19 5L16 8Z" fill="#9747FF" />
+		<G
+			stroke-width="1.5"
+			stroke-linecap="round"
+			stroke-linejoin="round"
```

Comment:
These values are producing React warnings -  `Warning: Invalid DOM property `stroke-width`. Did you mean `strokeWidth`?`

Reply:
Right, exactly 🙂  done in bd6e7f3ff7e2fc620d0f5c4d1ac060b459355a94